Microsoft.Toolkit.Uwp/Extensions/DispatcherQueueExtensions.cs

```diff
@@ -0,0 +1,251 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Did you git mv this file as I'm surprised it didn't mark it as a move...

Reply:
I just moved it as usual through Visual Studio, not sure why it wasn't picked up 🤔
Maybe it's because I also renamed the file in the process?

Reply:
You must move first [commit] and then rename. You can do both in one commit